components/post_view/channel_intro_message/index.ts

```diff
@@ -40,6 +41,9 @@ function mapStateToProps(state: GlobalState) {
         creatorName: getDisplayNameByUser(state, creator),
         teammate,
         teammateName: getDisplayNameByUser(state, teammate),
+        stats: getAdminAnalytics(state),
+        usersLimit: parseInt(getConfig(state).ExperimentalCloudUserLimit!, 10),
```

Comment:
Could we default this to 10 in case the `CloudUserLimit` is not set for on-prem installations?